pkgs/development/libraries/tracker-miners/default.nix

```diff
@@ -108,6 +106,7 @@ stdenv.mkDerivation rec {
   mesonFlags = [
     # TODO: tests do not like our sandbox
     "-Dfunctional_tests=false"
+    "-Dminer_rss=false"
```

Comment:
Maybe add a comment that it is unmaintained and insecure. Otherwise looks good, I was not able to find anything else than gnome-news depending on grss either.

Reply:
Will do, good point.